src/components/NcButton/NcButton.vue

```diff
@@ -430,6 +430,7 @@ export default {
 		display: inline-flex;
 		align-items: center;
 		justify-content: space-around;
```

Comment:
```suggestion
		justify-content: center;
```
Otherwise the wide button on the docs looks weird:
![grafik](https://user-images.githubusercontent.com/2496460/228255790-9f0e15cb-1649-40a9-acb7-5ff72f977e6a.png)

With this change it looks fine:
![grafik](https://user-images.githubusercontent.com/2496460/228256092-3721dea4-b6cb-4b36-9d95-ccf4f683b640.png)

Reply:
Great catch!